tdr-admins/tdr-user-administrator.md

```diff
@@ -61,11 +61,10 @@ If a new user needs to be added, then:
     * Email
 4. In the "Required User Actions" add the following options:  
   * Configure OTP: *(this will enforce MFA)*
-    * **Note**: For the moment, do not configure OTP (one-time password). The TDR team are currently working out which multi-factor authentication methods will be supported.
 5. Click "save"
 6. Under the "Credentials" tab: ![](images/tdr-user-administrator/adding_new_tb_user/users_3_v2.png)
 7. Request the user updates their password:
-  * Under the "Credentials Reset" section add the "Update Password (UPDATE_PASSWORD)" option to the "Reset Actions"
+  * Under the "Credentials Reset" section add the "Update Password (UPDATE_PASSWORD)" option and the "Configure OTP (CONFIGURE_TOTP)" option to the "Reset Actions"
   * Click the "Send Email" button. This will send an email to the user, with a URL link requesting they set a password
```

Comment:
This line needs updating to state: * Click the "Send Email" button. This will send ... requesting they **configure TOTP and** set a password"

Reply:
Makes sense, I've added it in.